tests/integration-tests/tests/common/utils.py

```diff
@@ -139,12 +139,13 @@ def retrieve_pcluster_ami_without_standard_naming(region, os, version, architect
 
 @retry(stop_max_attempt_number=3, wait_fixed=5000)
 def fetch_instance_slots(region, instance_type, multithreading_disabled=False):
-    instance_type_info = get_instance_info(instance_type, region)
-    vcpus = instance_type_info.get("VCpuInfo").get("DefaultVCpus")
-    default_threads_per_core = instance_type_info.get("VCpuInfo").get("DefaultThreadsPerCore")
-    if default_threads_per_core is None:
-        supported_architectures = instance_type_info.get("ProcessorInfo", {}).get("SupportedArchitectures", [])
-        default_threads_per_core = 2 if "x86_64" in supported_architectures else 1
+    vcpu_info = get_instance_info(instance_type, region).get("VCpuInfo", {})
+    vcpus = vcpu_info.get("DefaultVCpus")
+    default_threads_per_core = vcpu_info.get("DefaultThreadsPerCore")
+    if not vcpus:
+        raise Exception("'DefaultVCpus' can not be None.")
```

Comment:
what about 
`'DefaultVCpus' cannot be found in DescribeInstanceTypes API response`
?
or something that make clearer from where the value is missing from